.goreleaser.yml

```diff
@@ -1,5 +1,14 @@
+# Visit https://goreleaser.com for documentation on how to customize this
+# behavior.
+before:
+  hooks:
+    # this is just an example and not a requirement for provider building/publishing
```

Comment:
Let's remove this hook since it's just an example

Reply:
Done.